migrations/versions/bfcf4182dcb5_.py

```diff
@@ -0,0 +1,43 @@
+"""empty message
+
+Revision ID: bfcf4182dcb5
+Revises: 86c0f6b6a176
+Create Date: 2021-03-01 09:52:03.376234
+
+"""
+from alembic import op
+import sqlalchemy as sa
+
+from slugify import slugify
+
+
+# revision identifiers, used by Alembic.
+revision = "bfcf4182dcb5"
+down_revision = "86c0f6b6a176"
+branch_labels = None
+depends_on = None
+
+
+def upgrade():
+    conn = op.get_bind()
+    # ### commands auto generated by Alembic - please adjust! ###
+    op.add_column(
+        "organisations", sa.Column("slug", sa.String(length=255), nullable=True)
+    )
+    orgs = conn.execute("select name from organisations;")
+    for org in orgs:
+        name = org[0]
+        query = (
+            f"UPDATE organisations SET slug = '{slugify(name)}' WHERE name = '{name}';"
+        )
+        op.execute(query)
+    op.alter_column("organisations", "slug", nullable=False)
+    op.create_unique_constraint(None, "organisations", ["slug"])
```

Comment:
Without a name for the unique constraint, this will throw an error while removing the constraint on downgrade - because there exists more than unique constraint for the organisations table( one for the name and another for the slug). Please use  `organisations_slug_key` to be in line with another unique constraint `organisations_name_key`

Reply:
@ramyaragupathy thanks, I added it